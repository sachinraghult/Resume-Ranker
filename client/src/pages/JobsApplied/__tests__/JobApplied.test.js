import React from "react";
global.React = React;
import ReactDOM from "react-dom/client";
import JobApplied from "../JobsApplied";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";
import { act } from "react-dom/test-utils";
import { BrowserRouter } from "react-router-dom";

it("renders without crashing", async () => {
  const root = ReactDOM.createRoot(document.createElement("div"));
  await act(async () =>
    render(
      <BrowserRouter>
        <JobApplied />
      </BrowserRouter>
    )
  );
});

it("renders correctly", () => {
  render(
    <BrowserRouter>
      <JobApplied />
    </BrowserRouter>
  );
});

it("matches snapshot", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <JobApplied />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
