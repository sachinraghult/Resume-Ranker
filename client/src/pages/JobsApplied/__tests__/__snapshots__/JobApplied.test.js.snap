// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div>
    <div
      className="skin-blue-black"
    >
      <div
        className="wrapper"
      >
        <header
          className="main-header col-lg-12"
        >
          <span
            className="logo-mid first border"
            style={
              Object {
                "backgroundColor": "white",
                "paddingLeft": "2px",
              }
            }
          />
          <div
            className="fa fa-bars"
            id="menu_bar"
            onClick={[Function]}
          />
          <div
            className="user-profile"
            onClick={[Function]}
          >
            <span>
              user
            </span>
            <div
              className="mini-user fa fa-user-circle"
            />
          </div>
        </header>
        <div
          className="leftMenu openMenu"
        >
          <ul
            className="leftMenuList"
          />
        </div>
        <div
          className="content-wrapper content-wrapper-mid"
        />
        <div
          className="profile-hover"
          style={
            Object {
              "display": "none",
            }
          }
        >
          <div
            className="user-profile-icon fa fa-user-circle"
          />
          <div
            className="user-profile our-profile"
          >
            <span>
              user
            </span>
            <h1
              onClick={[Function]}
            >
              LogOut
            </h1>
          </div>
        </div>
        <div
          className="popup-overlay"
        />
        <div
          className="popup-overlay-bell"
        />
      </div>
    </div>
  </div>
  <div
    style={
      Object {
        "marginLeft": "50px",
      }
    }
  >
    <h3
      style={
        Object {
          "marginBottom": "25px",
          "marginLeft": "200px",
          "marginTop": "50px",
        }
      }
    >
      My Applications
    </h3>
    <div
      className="searchedjobs"
    >
      <div
        className="jobcards"
      >
        <h5
          style={
            Object {
              "marginLeft": "200px",
            }
          }
        >
          You haven't applied for any jobs yet
        </h5>
      </div>
    </div>
  </div>
</div>
`;
